docs/rules/prefer-array-index-of.md

```diff
@@ -0,0 +1,31 @@
+# Prefer `Array#indexOf()` over `Array#findIndex()` when searching the index of an item
+
+All built-ins have [`Array#indexOf()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) in addition to [`Array#findIndex()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex). Prefer `Array#indexOf()` over use of `Array#findIndex()` passing a function that tests equality as parameter
```

Comment:
All built-ins do not have an `.indexOf` method. I think this text needs more work.

Reply:
Is it better now?

Reply:
Is it OK now?